modules/031-local-path-provisioner/images/helper/werf.inc.yaml

```
@@ -1,28 +1,41 @@
 ---
 image: {{ $.ModuleName }}/{{ $.ImageName }}-artifact-dir-manager
 final: false
-from: {{ $.Images.BASE_GOLANG_20_ALPINE }}
-git:
-  - add: /{{ $.ModulePath }}modules/031-{{ $.ModuleName }}/images/{{ $.ImageName }}/dir-manager
-    to: /src
-    stageDependencies:
-      install:
-        - '**/*'
+from: {{ $.Images.BASE_GOLANG_23_ALPINE }}
+import:
+- image: {{ .ModuleName }}/{{ .ImageName }}-src-artifact
+  add: /src
+  to: /src
+  before: install
 shell:
-  installCacheVersion: "2023-10-16.2"
   install:
-    - export "CGO_ENABLED=0"
-    - export "GOOS=linux"
-    - export "GOARCH=amd64"
-    - cd /src
-    - echo "Build manager"
-    - GOPROXY={{ $.GOPROXY }} go build -o /tmp/manager main.go
-    - chmod 0777 /tmp/manager
+  - cd /src
+  - GOPROXY={{ $.GOPROXY }} CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o /manager main.go
+  - chmod 0777 /manager
```

`0777` is necessary?  may be  we can use `0700`?

```suggestion
  - chmod 0700 /manager
```

---

it can be a problem
```
helperPod.yaml: |-
    apiVersion: v1
    kind: Pod
    metadata:
      name: helper-pod
    spec:
      securityContext:
        runAsNonRoot: false
        runAsUser: 0
        runAsGroup: 0
      containers:
      - name: helper-pod
        image: {{ include "helm_lib_module_image" (list $context "helper") }}
        imagePullPolicy: IfNotPresent
      imagePullSecrets:
```
helper pod runs as root